### PR TITLE
Raspi sensor tests

### PR DESCRIPTION
**Pull request by SlupikB**
The code to be pulled includes following scripts:

- BME280 Sensor reader i2cpressure.py
- DS18B20 Sensor reader w1temperature.py
- Soft shutdown using button button_shutdown.py
This script runs on boot. To shutdown Pi hold down the Button for 10 seconds.
- led_button.py, led.py - blink led, pretty useless
- allsensors.py runs both sensors. Enter "python allsensors.py inf" to loop the readings.

The above are to be used with the test circuit you all know.
Please excuse lack of comments in the code, gonna add them on friday

### CODE REVIEW

**Comment by evemorgen** [comment]
Please set your editor to put 4 spaces in place of `TAB` character, GitHub renders tabs as 8 spaces which is not readable at all

File: code/raspi_sensor_tests/w1temperature.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import os
+from glob import glob
+from time import sleep
+
+os.system('modprobe w1-gpio')
```
[the remark this answers is not in the document]
Comment (evemorgen) [reply code/raspi_sensor_tests/w1temperature.py]: or some `setup.sh` script

File: code/raspi_sensor_tests/w1temperature.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+device_folder = glob(base_dir + '28*')[0]
+device_file = device_folder + '/w1_slave'
+
+def read_raw():
+	f = open(device_file)
+	lines = f.readlines()
+	f.close()
+	return lines
+
+def read_temp():
```
Comment (evemorgen) [inline code/raspi_sensor_tests/w1temperature.py]: there is a lib for it - [w1thermsensor](https://github.com/timofurrer/w1thermsensor)
parsing it manualy is kinda weird and error vulnerable

**Comment by evemorgen** [comment]
please provide `requirements.txt` file with all new dependencies needed, like `bme280`. Fixing lib version might be a good idea as well, like this `bme280==0.5`

File: code/raspi_sensor_tests/i2cpressure.py [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import smbus2
+import bme280
+from time import sleep
+
+i2c = smbus2.SMBus(1)
+# calibration_params = bme280.load_calibration_params(i2c, 0x76)
+
+
+data = bme280.sample(i2c, 0x76)
+print("GY-BMP280 Pressure: {p:.1f}\nGY-BMP280 Temp: {t:.1f}\n".format(p=data.pressure, t=data.temperature))
```
[the remark this answers is not in the document]
Comment (SlupikB) [reply code/raspi_sensor_tests/i2cpressure.py]: I have trouble installing python 3.6 on the pi. Is it that necessary? I broke the long line into two, but left the printing format as it was

File: code/raspi_sensor_tests/led.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+import RPi.GPIO as GPIO
```
[the remark this answers is not in the document]
Comment (SlupikB) [reply code/raspi_sensor_tests/led.py]: Is your advice to delete this file?

File: code/raspi_sensor_tests/allsensors.py [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+BME280_ADDR_IMU = 0x77
+
+gps_port_path = "/dev/serial0"
+gps_serial_port = serial.Serial(gps_port_path, baudrate = 9600, timeout = 0.5)
+
+a = time()
+
+while True:
+	sleep(0.1)
+	if not "-bme280" in sys.argv:
```
Comment (evemorgen) [inline code/raspi_sensor_tests/allsensors.py]: for now it's good enough, but please get familiar with [argparse](https://docs.python.org/3/howto/argparse.html) :)

File: code/raspi_sensor_tests/i2cpressure.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (evemorgen) [reply code/raspi_sensor_tests/i2cpressure.py]: it's not critical, leave it as it is ;)

**Comment by evemorgen** [comment]
Good job @SlupikB! Merging that right away

**Pull request merged**